sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py

```diff
@@ -48,29 +49,20 @@
 # How to command the arm and hand together
 robot_commander = SrRobotCommander(name="right_arm_and_hand")
 arm_commander.set_max_velocity_scaling_factor(0.1)
-
 rospy.sleep(rospy.Duration(2))
 
-# Start arm at home
-arm_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
-                        'ra_shoulder_lift_joint': -1.25, 'ra_wrist_1_joint': -0.733,
-                        'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': 0.00}
-rospy.loginfo("Moving arm to joint states\n" + str(arm_home_joints_goal) + "\n")
-robot_commander.move_to_joint_value_target_unsafe(arm_home_joints_goal, 6.0, True)
-
-# Move hand to open
-# Moving to a stored named target, stored targets can be viewed in MoveIt in the planning tab
-rospy.loginfo("Moving hand to joint state: open")
-hand_commander.move_to_named_target("open")
-
-rospy.sleep(3.0)
-
-# Move hand to pack
-# Moving to a stored named target, stored targets can be viewed in MoveIt in the planning tab
-rospy.loginfo("Moving hand to joint state: pack")
-hand_commander.move_to_named_target("pack")
-
-raw_input("Press Enter to continue...")
+# Start arm at home and hand at pack
+arm_hand_home_joints_goal = {'ra_shoulder_pan_joint': 0.00, 'ra_elbow_joint': 2.00,
+                             'ra_shoulder_lift_joint': -1.25, 'ra_wrist_1_joint': -0.733,
+                             'ra_wrist_2_joint': 1.5708, 'ra_wrist_3_joint': 0.00,
+                             'rh_THJ1': 0.52, 'rh_THJ2': 0.61, 'rh_THJ3': 0.0, 'rh_THJ4': 1.20,
+                             'rh_THJ5': 0.17, 'rh_FFJ1': 1.5707, 'rh_FFJ2': 1.5707, 'rh_FFJ3': 1.5707,
+                             'rh_FFJ4': 0.0, 'rh_MFJ1': 1.5707, 'rh_MFJ2': 1.5707, 'rh_MFJ3': 1.5707,
+                             'rh_MFJ4': 0.0, 'rh_RFJ1': 1.5707, 'rh_RFJ2': 1.5707, 'rh_RFJ3': 1.5707,
+                             'rh_RFJ4': 0.0, 'rh_LFJ1': 1.5707, 'rh_LFJ2': 1.5707, 'rh_LFJ3': 1.5707,
+                             'rh_LFJ4': 0.0, 'rh_LFJ5': 0.0, 'rh_WRJ1': 0.0, 'rh_WRJ2': 0.0}
+rospy.loginfo("Moving arm and hand to joint states\n" + str(arm_hand_home_joints_goal) + "\n")
+robot_commander.move_to_joint_value_target_unsafe(arm_hand_home_joints_goal, 6.0, True)
 
 # The arm commander generates a plan to a new pose before the pose is executed.
 # plan_to_pose_target(pose, end_effector_link, alternative_method, custom_start_state):
```

Comment:
maybe instead of putting huge comment here, just in a comment link method in arm commander? This ammount of comments really decreases code readability